mmhuman3d/utils/mesh_utils.py

```diff
@@ -88,11 +89,79 @@ def mesh_to_pointcloud_vc(
     return pointclouds
 
 
+def texture_uv2vc(meshes: Meshes) -> Meshes:
+    """Convert a Pytorch3D meshes's textures from TexturesUV to TexturesVertex.
+
+    Args:
+        meshes (Meshes): input Meshes.
+
+    Returns:
+        Meshes: converted Meshes.
+    """
+    assert isinstance(meshes.textures, TexturesUV)
+    device = meshes.device
+    vert_uv = meshes.textures.verts_uvs_padded()
+    batch_size = vert_uv.shape[0]
+    verts_features = []
+    num_verts = meshes.verts_padded().shape[1]
+    for index in range(batch_size):
+        face_uv = vert_uv[index][meshes.textures.faces_uvs_padded()
+                                 [index].view(-1)]
+
+        img = meshes.textures._maps_padded[index]
+        width, height, _ = img.shape
+
+        face_uv = face_uv * torch.Tensor([width - 1, height - 1
+                                          ]).long().to(device)
+
+        face_uv[:, 0] = torch.clip(face_uv[:, 0], 0, width - 1)
+        face_uv[:, 1] = torch.clip(face_uv[:, 1], 0, height - 1)
+        face_uv = face_uv.long()
+        faces = meshes.faces_padded()
+        verts_rgb = torch.zeros(1, num_verts, 3).to(device)
+        verts_rgb[:, faces.view(-1)] = img[height - 1 - face_uv[:, 1],
+                                           face_uv[:, 0]]
+        verts_features.append(verts_rgb)
+    verts_features = torch.cat(verts_features)
+
+    meshes = meshes.clone()
+    meshes.textures = TexturesVertex(verts_features)
+    return meshes
+
+
+def load_objs_as_meshes(files: List[str],
+                        device: Optional[Union[torch.device, str]] = None,
+                        load_textures: bool = True,
+                        **kwargs) -> Meshes:
+    if not isinstance(files, list):
+        files = [files]
+    return _load_objs_as_meshes(
+        files=files, device=device, load_textures=load_textures, **kwargs)
+
+
+def load_plys_as_meshes(
+    files: List[str],
+    device: Optional[Union[torch.device, str]] = None,
+    load_textures: bool = True,
+) -> Meshes:
+    writer = IO()
+    meshes = []
+    if not isinstance(files, list):
+        files = [files]
+    for idx in range(len(files)):
+        assert files[idx].endswith('.ply'), 'Please input .ply files.'
+        mesh = writer.load_mesh(
+            path=files[idx], include_textures=load_textures, device=device)
+        meshes.append(mesh)
+    meshes = join_meshes_as_batch(meshes, include_textures=load_textures)
+    return meshes
+
+
 def save_meshes_as_plys(meshes: Meshes = None,
                         verts: torch.Tensor = None,
                         faces: torch.Tensor = None,
                         verts_rgb: torch.Tensor = None,
-                        paths: List[str] = []) -> None:
+                        files: List[str] = None) -> None:
```

Comment:
For a saving method, dst path(files) should never be None.

Reply:
ok, but `[]` should not be used as a default value, maybe use `()`